frontend/src/lib/constants.tsx

```diff
@@ -87,6 +87,30 @@ export const MATHS: Record<string, any> = {
         onProperty: false,
         type: EVENT_MATH_TYPE,
     },
+    weekly_active: {
```

Comment:
@paolodamico What's the best way to feature flag this?

Also, could use some suggestions on how to best convey the difference between weekly active, monthly active vs unique users. Unique users will count uniques dependent on the interval provided in the time settings. But weekly active and monthly active will strictly do as said disregarding the interval. Should I change it to somehow be dependent on interval setting too?

Reply:
As we talked about, `preflight?.ee_enabled` + a feature flag.

I think the descriptions are okay for now (though if @paolodamico has better suggestions..), can iterate on this if we see people are confused